fixtures/workers-with-assets/README.md

```diff
@@ -0,0 +1,23 @@
+# workers-with-assets
+
+`workers-with-assets` is a test fixture that showcases Workers with Assets. This particular fixture sets up an assets-only Workers project.
+
+## dev
+
+To start a dev session you can run
+
+```
+wrangler dev
+```
+
+or
+
+```
+wrangler dev --name=assets-worker --experimental-assets=public
```

Comment:
nit: can we use `--experimental-assets=./public` in our examples to clearly indicate the value is a filepath?

Reply:
good callout! fixed ✅